components/TrackingOrderPage/TrackingOrderPage.tsx

```diff
@@ -107,14 +102,14 @@ const TrackingOrderPage = () => {
                   onClick: () => {
                      setDialogIsOpen(false)
                      router.push('/track_order')
-                  }
-               }
+                  },
```

Comment:
Avoid making unnecessary changes. Changes should have reasons to be made, that's why git keeps the history of everything, so they can be referred to later on if needed.